src/player/watch_media.rs

```diff
@@ -11,25 +10,43 @@ use crate::{
     media::Media,
     player::mpv::open_mpv,
     player::vlc::open_vlc,
-    TRANSLATION, USE_MPV,
+    TRANSLATION, USE_GECKODRIVER, USE_MPV,
 };
 
 #[tokio::main]
 pub async fn watch_media(media: Media, img_mode: Option<bool>) -> WebDriverResult<()> {
     let language = TRANSLATION.get().unwrap();
     let use_mpv = USE_MPV.get().unwrap();
+    let use_geckodriver = USE_GECKODRIVER.get().unwrap();
 
     let url = format!("https://vizer.in/{}", &media.url);
-    let mut chromedriver = Command::new("chromedriver").spawn().unwrap();
-    // we need to wait chromedriver to start :(
+    let mut command_name = String::new();
+    if *use_geckodriver {
+        command_name.push_str("geckodriver");
+    } else {
+        command_name.push_str("chromedriver");
+    };
+
+    let mut command = Command::new(command_name)
```

Comment:
Instead of `command` name it `browser_driver`

Reply:
Done :)